pr-data.csv

```diff
@@ -2884,7 +2884,7 @@ https://github.com/apache/incubator-shenyu,3777209fc76800175df3962213ca1febe57ed
 https://github.com/apache/incubator-streampipes,7ba9213012b72ea70edc6848c18ba6d90ff2c534,streampipes-connect,org.apache.streampipes.connect.adapter.format.json.xml.XmlTest.parseEventCarPark,ID,Accepted,https://github.com/apache/incubator-streampipes/pull/59,
 https://github.com/apache/incubator-streampipes,7ba9213012b72ea70edc6848c18ba6d90ff2c534,streampipes-connect,org.apache.streampipes.connect.adapter.format.json.xml.XmlTest.parseEventParkingFaciltyStatus,ID,Accepted,https://github.com/apache/incubator-streampipes/pull/59,
 https://github.com/apache/incubator-uniffle,6fb2a9a63132ab9abbde2f1f9240f9caf2f5d0f0,common,org.apache.uniffle.common.rpc.GrpcServerTest.testGrpcExecutorPool,TD,,,
-https://github.com/apache/incubator-wayang,b7b6c1ebf65b58f5c03ac2b9fd4c1846ea877d01,wayang-platforms.wayang-graphch,org.apache.wayang.graphchi.operators.GraphChiPageRankOperatorTest.testExecution,ID,,,
+https://github.com/apache/incubator-wayang,b7b6c1ebf65b58f5c03ac2b9fd4c1846ea877d01,wayang-platforms.wayang-graphch,org.apache.wayang.graphchi.operators.GraphChiPageRankOperatorTest.testExecution,ID,MovedOrRenamed,,https://github.com/apache/incubator-wayang/tree/8e8897900921e52283c1c6093e986e51a3737393
```

Comment:
Change the link to have `/commit/` instead of `/tree/`. If this commit changed the test, can you check if the test was `ID` and add a new line with the new name/location?